### PR TITLE
[HDRP][Template] Upgrade HDRP Template to 2023.2.0a21, bump template to 16.0.3, reconfigure Global Volume

### PR DESCRIPTION
**Pull request by arttu-peltonen**
### Purpose of this PR

This PR does two things:

1: Upgrades project version and all dirty files to 2023.2.0a21 (commit https://github.com/Unity-Technologies/Graphics/pull/7920/commits/ae454aae05d704238de0f15c11acd58e7ce01b6b)
- Committed files that were dirtied as a result of opening the project with 2023.2.0a21 and saving it (wxceptions: packages.lock file and anything under UserSettings were not added)
- The upgrade to 2023.2 is necessary because Default Volume Profile changes are only in 23.2. Therefore this PR should not be backported either.

2: Updates to HDRP Template required to configure Default Volume Profile correctly (commit https://github.com/Unity-Technologies/Graphics/pull/7920/commits/ae52d9104f6419aece2a09c424c16195140969bf)
- Removed Global Volume object from scene. This was a misconfiguration, since it was referencing the `VolumeGlobal.asset` profile, which was also referenced by Default Volume Profile inside HDRP Global Settings. This resulted in all overrides in this profile being applied twice every frame for no reason.
- Updated the `VolumeGlobal.asset` to contain all overrides (triggered by opening HDRP Global Settings). This is the new way the default volume profile works, it contains all components and all overrides, so that users can discover and modify all default values in HDRP Global Settings (see [VolumeManager Default Settings Management PR](https://github.cds.internal.unity3d.com/unity/unity/pull/20254))

---
### Testing status
- Tested that everything looks right in playmode when using the template.

---

### CODE REVIEW

File: com.unity.template-hd/Packages/com.unity.template.hd/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Changed
 Changes a few objects to no longer use lightmaps (bamboo floor, bamboo cage, long table 3rd room, column 3rd room)
+Removed Global Volume from scene (Default Volume Profile is now configured only through HDRP Global Settings)
```
Comment (sebastienlagarde) [inline com.unity.template-hd/Packages/com.unity.template.hd/CHANGELOG.md:11]: you need to bump the version to 16.0.3 here, as we just published 16.0.2 one week ago

**Review by sebastienlagarde — APPROVED** [review]
Please bump package version to 16.0.3 (don't forget the Exception validation) and we are good

**Review by iM0ve — CHANGES_REQUESTED** [review]
I think if we remove the volume from the hierarchy we should then also update the tutorial file. Because right now it guides users to select the global volume in the scene. I think only that one page and the first sentence in the next one should be changed.

image

**Comment by pierre-unity** [comment]
The following page of the tutorial will require a slight rewrite:

image

- Change 1: Explain that it's recommended from a performance point of view to use the global volume setting in the Project Settings.
-  Change 2: Tell the user to go into the Project Settings, and change a property of the global volume, so that they can see it affects the entire scene, regardless of camera position (tone mapping, or saturation for instance). This means updating the step 1, 2 and 3 on the page. And step 4 might need a slight change.

**Comment by pierre-unity** [comment]
Remaining tasks:
- Minor: Add a new page to the Tutorial regarding Global Volumes (a new Global Volume with only Fog, Sky and Shadows was added recently)
- Minor: rename the "Adpative Probe Volume" in the hierarchy to just "Probe Volume"


Done already:
- Moved Raytracing Volume profiles under the Qality > HDRP > Volume section
- Updated the doc for the Default Volume
- Polished the Probe Volumes tutorials

**Comment by arttu-peltonen** [comment]
According to @pierre-unity's task list I added the following page: 
image

File: com.unity.template-hd/Assets/SampleSceneAssets/Particles/FallingLeaves.vfx [hunk 2]
```diff
@@ -540,7 +540,7 @@ MonoBehaviour:
   m_Name: 
   m_EditorClassIdentifier: 
   m_UIIgnoredErrors: []
-  m_Parent: {fileID: 114350483966674976}
+  m_Parent: {fileID: 0}
```
Comment (sebastienlagarde) [inline com.unity.template-hd/Assets/SampleSceneAssets/Particles/FallingLeaves.vfx:543]: @arttu-peltonen  Looks like various FX have lose a fileID, wondering if the FX are still working with your change? poking @JMargevics  maybe Jans you could check if VFX is still ok in the template? thanks

**Comment by sebastienlagarde** [comment]
waiting for a quick QA valiadtion, poked Jans for it, then we are good to go

File: com.unity.template-hd/Assets/SampleSceneAssets/Particles/FallingLeaves.vfx (hunk 2 above)
Comment (arttu-peltonen) [reply com.unity.template-hd/Assets/SampleSceneAssets/Particles/FallingLeaves.vfx:543]: yeah I'm not sure why this happened - I tried reverting these but when I save the project, it happens again. I still see falling leaves & flying butterflies so it looks like it's fine.

Comment (iM0ve) [reply com.unity.template-hd/Assets/SampleSceneAssets/Particles/FallingLeaves.vfx:543]: I checked all VFX still work (Butterflies, dust, leaves)

**Review by iM0ve — APPROVED** [review]
What I tested:
* All VFX particles still work
* Went through the updated Volume tutorial
* Checked different Quality levels (Medium, High, RTX)
* Used 3rd person controller
* Checked that the cinematics still work.
* No warnings on open.

WxvSjaJrbT

**Comment by arttu-peltonen** [comment]
@sebastienlagarde this should be good to go

**Pull request merged**